llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -690,9 +690,12 @@ class InnerLoopAndEpilogueVectorizer : public InnerLoopVectorizer {
       const TargetTransformInfo *TTI, AssumptionCache *AC,
       OptimizationRemarkEmitter *ORE, EpilogueLoopVectorizationInfo &EPI,
       LoopVectorizationCostModel *CM, BlockFrequencyInfo *BFI,
-      ProfileSummaryInfo *PSI, GeneratedRTChecks &Checks, VPlan &Plan)
+      ProfileSummaryInfo *PSI, GeneratedRTChecks &Checks, VPlan &Plan,
+      bool ForMainLoop)
```

Comment:
Nit, if the only two callers of this constructor are `EpilogueVectorizerMainLoop` and `EpilogueVectorizerEpilogueLoop` below, should we just add arguments for VF/MinProfitableTripCount/UnrollFactor and pass them instead of the bool?